ts/features/payments/checkout/components/WalletPaymentFailureDetail.tsx

```diff
@@ -51,6 +53,13 @@ const WalletPaymentFailureDetail = ({ failure }: Props) => {
     supportModal.present();
   };
 
+  const handleChangePaymentMethod = () => {
+    dispatch(paymentsCalculatePaymentFeesAction.cancel());
+    navigation.replace(PaymentsCheckoutRoutes.PAYMENT_CHECKOUT_NAVIGATOR, {
+      screen: PaymentsCheckoutRoutes.PAYMENT_CHECKOUT_MAKE
+    });
```

Comment:
To potentially improve the user experience, rather than using `navigation.replace`, it might be helpful to use `navigation.pop()` instead. This approach displays the previous screen in the navigation stack, performing a leftward transition animation as if the user is going back. Let me know what do you think about.